### PR TITLE
Open letter text for companies

### PR DESCRIPTION
**Pull request by weaverbel**
Is this good to go now?

### CODE REVIEW

File: _posts/2018/01/2018-01-17-sponsor-carpentrycon.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Attendees will be a mixture of early career researchers and more senior academics, so the potential is there for great conversations 
+with people about how they can best leverage the services you offer to solve complex research problems. 
+
+Want to help foster research workforce diversity? We plan to provide travel scholarships for people to attend from the Global South. 
+Workshops and skill-ups will focus not just on tools and skills but on fostering attitudes of openness, diversity, inclusivity 
+and reproducibility, because those are just as essential skills and tools to research success.
+
+### Who or what are the Carpentries?
+
+CarpentryCon 2018 is a joint initiative of [Software Carpentry](https://software-carpentry.org/) and [Data Carpentry](http://www.datacarpentry.org/) (The Carpentries). The Carpentries are sibling, non-profit projects that teach foundational computational and data science skills to researchers. We have taught more than 34,000 researchers worldwide.
```
Comment (tracykteal) [inline _posts/2018/01/2018-01-17-sponsor-carpentrycon.md:52]: Now that we're The Carpentries, we can say "CarpentryCon 2018 is an initiative of The Carpentries, the joint project of Software Carpentry and Data Carpentry. The Carpentries is a non-profit project that teaches..."

File: _posts/2018/01/2018-01-17-sponsor-carpentrycon.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+But to innovate, researchers need more. Our  people want  to hear about tools or services that can take their 
+research to the next level. 
+
+*So there is real synergy there between your organisation's goals and ours.*
+
+CarpentryCon offers a great opportunity to put what you offer in front of a highly motivated and receptive global audience. 
+Attendees will be a mixture of early career researchers and more senior academics, so the potential is there for great conversations 
+with people about how they can best leverage the services you offer to solve complex research problems. 
+
+Want to help foster research workforce diversity? We plan to provide travel scholarships for people to attend from the Global South.
```
Comment (tracykteal) [inline _posts/2018/01/2018-01-17-sponsor-carpentrycon.md:46]: Scholarships will be for more than that region. Could it be instead "Want to help foster workforce diversity and bring new communities into computing? We plan to provide travel scholarships for people from many geographies and communities to participate, including the global South."

File: _posts/2018/01/2018-01-17-sponsor-carpentrycon.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+The Carpentries teach introductory material - enough to get people started with coding and data science tools. 
+But to innovate, researchers need more. Our  people want  to hear about tools or services that can take their 
+research to the next level. 
+
+*So there is real synergy there between your organisation's goals and ours.*
+
+CarpentryCon offers a great opportunity to put what you offer in front of a highly motivated and receptive global audience. 
+Attendees will be a mixture of early career researchers and more senior academics, so the potential is there for great conversations 
+with people about how they can best leverage the services you offer to solve complex research problems.
```
Comment (tracykteal) [inline _posts/2018/01/2018-01-17-sponsor-carpentrycon.md:44]: While this is for sponsorship, participants will also be reading this. Something that's not in the above section very much is about bringing together 'instructors' and having us learn from each other how to teach better. Could we add something about this in the 'Why Sponsor' and 'What is CarpentryCon about' sections?

File: _posts/2018/01/2018-01-17-sponsor-carpentrycon.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+authors: ["Belinda Weaver"]
+title: "An Open Letter to Companies - Sponsor CarpentryCon 2018"
```
Comment (tracykteal) [inline _posts/2018/01/2018-01-17-sponsor-carpentrycon.md:4]: This post is great and relevant to any organization that would want to support CarpentryCon. Could the title be: "Why Become a CarpentryCon Sponsor?" like for PyCon https://us.pycon.org/2017/sponsors/why-sponsor/

**Comment by tracykteal** [comment]
I made a few comments. We should probably also include in the post what it is that Sponsorship lets us do, like fund travel scholarships. The PyCon 2017 Sponsor post seems to have some good ideas https://us.pycon.org/2017/sponsors/why-sponsor/  Also, I think we have our 'levels' established, essentially our Sponsor prospectus? Can we link to that from this post?

**Comment by weaverbel** [comment]
@tracykteal I have actually significantly reworked this post after reading your comments and looking at the PyCon page, so I might close this PR and get comments on the new post.

**Pull request closed without merging**